test/integration/processing/settlement/get-last-settlement.test.js

```diff
@@ -0,0 +1,101 @@
+const db = require('../../../../app/data')
+
+const { getLastSettlement } = require('../../../../app/processing/settlement')
+const { SETTLEMENT } = require('../../../mock-components/mock-dates')
```

Comment:
Can we use the same format we do in the mock objects of importing a specific 'type' of SETTLEMENT date?

i.e.  `const { DATE: SETTLEMENT_DATE } = require('../../../mock-components/mock-dates').SETTLEMENT`